src/coreclr/scripts/superpmi.py

```diff
@@ -1907,12 +1919,12 @@ async def create_one_artifact(jit_path: str, location: str, flags: list[str]) ->
                     for item in self.diff_mcl_contents:
                         diff_items.append(item)
 
-                    logging.info("Creating dasm files")
+                    logging.info("Creating dasm files: %s %s", base_asm_location, diff_asm_location)
```

Comment:
As we discussed offline, also display the exact `jit-analyze` command line to invoke?

Reply:
Optional, if running asmdiff for private store, then pass `-count 0` to the `jit-analyze` command line so it will just show the summary instead of other details?

Reply:
There's no private store yet in the script (that change isn't committed). I'm not in favor of this, since you'll end up manually running jit-analyze to see the data you want to see, anyway.